setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
     onc
     sqlalchemy
     spans
+    wget
```

Comment:
Let's use `requests`, not `wget`.  `wget` seems to be dead content as it's last update was 2015 and it's [bitbucket link is daed](https://pypi.org/project/wget/)

tl;dr: replace `wget.download` with `requests.get`

Reply:
wget is a far more basic command, simply a wrapper on the Linux wget command, plus easier usage in Python.

Are you sure you want to make this change?

Reply:
yes.  `requests` is the standard here.  Need Windows and Mac compliance.  Python wget wrapper doesn't have any documentation, so I don't know how to pass any of the dozen `wget` command line arguments via python.